### DISCUSSION TITLE
Boxplot statistics

### DISCUSSION THREAD

**Discussion by NickysQ** [hnolCol/instantclue#54] [Q&A]
We recently started using Instant Clue (Windows version). Really cool tool. Thanks for sharing.
Q: What stats are shown in the default box plot graphs? According to Matplotlib settings it is the median but the summary data only gives the mean? Are the wiskers reflecting the standard deviation?

Also:
In the Windows version it is not possible to add text to the Main Figure, is that correct?
Is there a possibility to change the font size of the axis label (a, b, etc) in the Main Figure?

Thank you!

**Comment by hnolCol** [comment]
Hi, 

thank you very much for your feedback. 
**1) Add text to main figure**
In the settings -> Boxplot. (Settings is the icon in the very bottom right corner) you can find the respective options for boxplot.
![image](https://user-images.githubusercontent.com/24191910/159909828-09156d61-747a-48f5-b4dd-50843967f1ba.png)

The line in the box is the median (in the summary data it is given by "50%" instead of the name median (I will replace that with "median (50%)") representing the 50% quantile. The upper and lower limit show the 25% and 75% quantiles. The whiskers do not show the standard deviation but rather display the max and min values. Which is by default 1.5 times the inter quantile range (height of the box => 75% - 25% = IQR. The setting (see img above gives the option "boxplot.whiskers" which defaults to 1.5. I have plotted some tutorial data with different settings for the boxplot.whisker setting. 
![image](https://user-images.githubusercontent.com/24191910/159910596-9dead23c-946a-478c-9681-310dd15bf353.png)
The outliers (e.g. crosses) are defined by this setting. Assume you have the boxplot.whisher setting at 1.5. Then all the data that are further away from the median than 1.5 * IQR are considered as "outliers". (e.g. outside the whiskers). I hope that this helps. :) The settings also allow to show the mean if you would like to do so. 
Let me know if you need more details. 

**2) Add text to main figure**

Yes at the moment that is correct, it will change in the future and will be available again. 

**3) Axis label size**

I have added an option to the setting to alter the size of the label in the main.figure. Which will be added to the version 0.11.2.
See below:
![image](https://user-images.githubusercontent.com/24191910/159909362-ca7964ce-3b51-4218-ae01-e93cdc741139.png)

Thank you very much for your input.
